app/model/song.js

```diff
@@ -0,0 +1,30 @@
+var express = require('express');
+var app = express();
+var fs =require('fs');
+
+Song = function() {
+ this.songs = [];
+ this.loadAll();
+}
+
+Song.prototype.get =function(id) {
+  if (!id) {return false;}
+  return this.songs[id];
+}
+
+Song.prototype.loadAll = function(callback) {
```

Comment:
This behavior looks repeated from the artists.js and albums.js, consider refactoring to some explicit module and importing it into artist, song and album.